docs/content/tutorials/tutorial-delete-data.md

```diff
@@ -50,27 +50,22 @@ Permanent deletion of a Druid segment has two steps:
 1. The segment must first be marked as "unused". This occurs when a segment is dropped by retention rules, and when a user manually disables a segment through the Coordinator API. This tutorial will cover both cases.
```

Comment:
Please remove the mention of 'retention rules' here, as well as the reference to "Tutorial: Configuring retention" being 'highly recommended' above. (We shouldn't be using retention rules anymore after these edits)

Reply:
sure, removed all references to 'retention' rules in this tutorial.